minimint/src/lib.rs

```diff
@@ -145,6 +173,89 @@ impl MinimintServer {
         }
     }
 
+    // Build a `ConsensusOutcome` then use the API to validate and process missed epochs
+    pub async fn rejoin_consensus(
+        &mut self,
+        _rng: &mut (impl RngCore + CryptoRng + Clone + 'static),
+    ) {
+        // Send a `RejoinRequest` message to all peers
+        self.connections
+            .send(
+                Target::AllExcept(BTreeSet::new()),
+                EpochMessage::RejoinRequest,
+            )
+            .await;
+
+        // Advance consensus to the max(next_epoch) from max_evil + 1 peers
```

Comment:
I was hoping to avoid something like a safety margin since it's kind of hacky.

I think what I'll do instead is something I mentioned earlier - submit the rejoin request as a consensus item.  Then all the honest peers can flush their send buffers and begin sending hbbft messages at the next epoch.